### ISSUE TITLE
Scale

### ISSUE THREAD

**Issue by Arb0l** [teuteuf/worldle#306]
Use of Mercator distorts the shape of large countries such as Russia. Using Peters projection would be more realistic, especially when only used for individual countries.

**Comment by ke4roh** [comment]
Agreed. Mercator has its uses, but there are better projections for individual country display. Canada's northern islands look huge! https://www.esri.com/arcgis-blog/products/arcgis-pro/mapping/mercator-its-not-hip-to-be-square/

**Comment by Arb0l** [comment]
I’m afraid today’s example is an excellent example of why Mercator just isn’t suitable for this otherwise wonderful game. A much abused term, but sadly it renders the game ‘not fit for purpose’. Please consider a more  credible way of representing countries according to their land area!
________________________________

**Comment by jimbali** [comment]
Yeah Alaska is looking pretty monstrous today 😆
